Sage/carbon
# docs(flat-table): fix accessibility violations - FE-4255

### Proposed behaviour

Fixes accessibility violations in `flat-table-expandable--sizes` story

### Checklist




#### QA


### Testing instructions
See `flat-table-expandable--sizes` story and confirm that there are no accessibility violations
